tests/Event/Http/Fixture/ag-v1-query-string-multivalue.json

```diff
@@ -15,10 +15,14 @@
         "X-Forwarded-Proto": "https"
     },
     "queryStringParameters": {
-        "foo": "baz"
+        "foo": "bar"
```

Comment:
Any reason for this change?

I "think" it was `baz` because (I think) API Gateway takes the last value of the array

Reply:
Pretty sure that when my changes got put in, this the data in the JSON would've failed the test if it was kept -- and from memory only the abc[] vars had multi-vars, not the 'foo' variant (eg. foo[] would have multi-vars, foo not). The 'shapes[]' inclusion below it tested multi-vars, and I went into much more detail later on for the FpmHandler